src/components/math-spinner.tsx

```diff
@@ -0,0 +1,105 @@
+'use client'
+
+import { useState } from 'react'
+
+function SimpleMathSpinner() {
```

Comment:
Nesse arquivo, invés de usar export default... e só declarar function SimpleMathSpinner(), faz assim

export function SimpleMathSpinner() {

exclui a linha de export default e no arquivo que ta importando o SimpleMathSpinner, inves de import SimpleMathSpinner coloca import { SimpleMathSpinner }.

explicação do porque: 

https://chatgpt.com/share/6769c66e-20d8-8006-b820-70a367019adf

perdão to mt exausto <3